apps/machine/zynqmp/platform_info.h

```diff
@@ -51,6 +51,63 @@ struct remoteproc_priv {
 #define POLL_STOP 0x1U
 #endif /* RPMSG_NO_IPI */
 
+#define RPU_CPU_ID          0 /* RPU remote CPU Index. We only talk to
```

Comment:
where is this used? Can we use XPAR_CPU_ID ?

Reply:
XPAR_CPU_ID is not provided in linux.